cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -356,6 +356,9 @@ data:
   # Whether to start the discovery daemon to watch for raw storage devices on nodes in the cluster.
   # This daemon does not need to run if you are only going to create your OSDs based on StorageClassDeviceSets with PVCs.
   ROOK_ENABLE_DISCOVERY_DAEMON: "false"
+  # Enable volume replication controller
```

Comment:
comment out these new fields in both operator.yaml and openshift-operator.yaml

Reply:
https://github.com/rook/rook/pull/7403#discussion_r594561531 
This comment says otherwise.